smart-py/marketplace.py

```diff
@@ -6,8 +9,11 @@ def __init__(self, objkt, metadata, manager, fee):
             manager = manager,
             swaps = sp.big_map(tkey=sp.TNat, tvalue=sp.TRecord(issuer=sp.TAddress, objkt_amount=sp.TNat, objkt_id=sp.TNat, xtz_per_objkt=sp.TMutez, royalties=sp.TNat, creator=sp.TAddress)),
             counter = 500000,
-            fee = fee
-            )
+            fee = fee,
+            fee_account = manager,
+            secondary_accounts = sp.set([manager]),
+            last_ping = sp.timestamp(0)
```

Comment:
Ideally one would use here sp.now, instead of sp.timestamp(0)
I had problems to run the tests when i used sp.now. I think is more a problem with the test: there must be a way to define now in the contract constructor as one does in the run method of the entrypoints.
Anyone knows how to fix it?